s4e-web/cypress/page-objects/map/map-locations-search.po.ts

```diff
@@ -60,17 +59,17 @@ export class LocationsSearch extends Core {
       .should('contain', voivodeship);
 
     return LocationsSearch;
-  }
+  };
 
   static selectNthResult(nth: number) {
     LocationsSearch
       .pageObject
       .getSearchResults()
       .eq(nth)
-      .click({ force: true });
+      .click();
 
     return LocationsSearch;
-  }
+  };
```

Comment:
Semicolon after methods isn't required. Please remove it due to consistency in code (not only cypress files).